backend/src/apiserver/server/pipeline_server.go

```diff
@@ -573,6 +573,94 @@ func NewPipelineServer(resourceManager *resource.ResourceManager, options *Pipel
 	return &PipelineServer{resourceManager: resourceManager, httpClient: http.DefaultClient, options: options}
 }
 
+// Creates a pipeline and a pipeline version in a single transaction.
+// Supports v2beta1 behavior.
+func (s *PipelineServer) CreatePipelineAndVersion(ctx context.Context, request *apiv2beta1.CreatePipelineAndVersionRequest) (*apiv2beta1.Pipeline, error) {
```

Comment:
Although the resources created here include both pipeline and pipeline version, we only return the pipeline. Should we return both of them here?

Reply:
Thank you. This is to be consistent with `UploadPipeline`, which also returns a pipeline. We can consider changing the API to be closer to AIP requirements, but I would rather leave it to post-GA.